README.md

```diff
@@ -1,6 +1,6 @@
 # amiapp_flutter
 
-Sample Ami app in Flutter
+Runner app in Flutter
```

Comment:
I know the iOS project is named Runner in Flutter, but I think Ami app was still better here, no? 😅

Reply:
it indeed was, let me change it :D